xep-0308.xml

```diff
@@ -100,11 +106,17 @@
   <p>While it's not possible to prevent this protocol from being used in such a way, it is not intended that it provides a way to continue expanding a previous message indefinitely and clients, in as much as it is sensible, should present use of this extension only for correction rather than for providing a continuous stream, for which &xep0301; can be used instead.</p>
   <p>Correction MUST only be used to change the details of a stanza (e.g. the message body) and not to change the nature of the stanza (e.g. correction MUST NOT be used to turn a chat message into a pubsub notification)</p>
   <p>While it is possible to use this protocol to correct messages older than the most recent received from a full JID, such use is out of scope for this document and support for this SHOULD NOT be assumed without further negotiation.</p>
+  <p>A &xep0045; room SHOULD reject any correction attempt made by a different real bare JID than the one from which the message originates.  In order to notify clients of this behaviour, it MUST include the 'urn:xmpp:message-correct:0' feature in its disco#info if it applies this rule.</p>
```

Comment:
Why SHOULD and not MUST?
Also, does the reverse hold true - must a room always allow correction when the bare jids are the same?